.github/workflows/sonarqube.yml

```diff
@@ -8,10 +8,12 @@ on:
 
 jobs:
   main:
-    runs-on: ubuntu-latest
+    runs-on: zendesk-stable
```

Comment:
@hansol-kim-zendesk, I _think_ the diff we got was borked, because [this shouldn't work](https://zendesk.slack.com/archives/CV1GMGC0H/p1613352547158400).

Indeed, [what was configured before is the recommended setting](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources).